winlogbeat/beater/eventlogger.go

```diff
@@ -178,14 +178,22 @@ runLoop:
 			if eventlog.IsRecoverable(err) {
 				e.log.Warnw("Read() encountered recoverable error. Reopening handle...", "error", err, "channel", api.Channel())
 				if closeErr := api.Close(); closeErr != nil {
-					e.log.Warnw("Close() error.", "error", err)
+					e.log.Warnw("Close() error for channel %q.", "error", closeErr, "channel", api.Channel())
```

Comment:
`Warnw` doesn't accept any printf style format strings.

```suggestion
					e.log.Warnw("Close() error, "error", closeErr, "channel", api.Channel())
```

Reply:
thanks.  fixed and made the content the same for the 3 error messages.